src/interface/blas3/backend/default_cpu.hpp

```diff
@@ -94,7 +94,27 @@ typename executor_t::policy_t::event_t _gemm(
 
 #endif
 }
+
+
+
```

Comment:
That's a lot of new empty space. Wouldn't clang-format fix that?

Reply:
Already removed